packages/client/utils/retroGroup/updateClonePosition.ts

```diff
@@ -25,22 +52,56 @@ const getTransition = (isClipped: boolean, timeRemaining: number) => {
 }
 
 export const getDroppingStyles = (
-  targetEl: HTMLDivElement,
+  element: HTMLDivElement,
   bbox: ClientRect,
   maxTop: number,
-  timeRemaining: number
+  timeRemaining: number,
+  targetId?: string | null,
+  isClose?: boolean,
+  lastZIndex?: number
 ) => {
   const spotlightEl = document.getElementById('spotlight')
-  const showAboveSpotlight = spotlightEl ? spotlightEl.contains(targetEl) : false
+  const isTargetInSpotlight = targetId
+    ? !!(spotlightEl && document.querySelector(`div[${DragAttribute.DROPPABLE}='${targetId}']`))
+    : false
+  const isInSpotlight = spotlightEl && spotlightEl.contains(element)
+  const showAboveSpotlight = isInSpotlight || isTargetInSpotlight
   const {top, left} = bbox
-  const minTop = getMinTop(top, targetEl)
+  const minTop = getMinTop(top, element)
   const clippedTop = Math.min(Math.max(minTop, top), maxTop - bbox.height)
   const isClipped = clippedTop !== top
+
+  const fadeInAnimation = `${reflectionSpotlightFadeIn.toString()} 0.5s linear 0s forwards`
+  const fadeOutAnimation = `${reflectionSpotlightFadeOut.toString()} 0.5s linear 0s forwards`
+
+  const notInSpotlightSuggestions = spotlightEl && !isInSpotlight
```

Comment:
It looks like `isInSpotlight` already checks whether `spotlightEl` exists so we can remove this line and just use `isInSpotlight`

Reply:
@nickoferrall There is a small difference — in the second case, we check that the element is not in the spotlight **but** spotlight is open. It could be that spotlight is closed and therefore the element is not in the spotlight.
Changed the code and comments a bit. Can you check if it is more clear now?

Reply:
That makes sense - thanks!